templates/base.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>{{ title }}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- Genearl Stylesheets -->
+    <link rel="stylesheet" href="/static/styles/eventslyStyleSheet.css">
+    <link rel="stylesheet" href="/static/styles/navbarStyleSheet.css">
+    <link rel="stylesheet" href="/static/styles/footerStyleSheet.css">
+
+    <!-- Specific Styles NOTE: (variable shows as error, but it is not)-->
+    <style type="text/css">{{ pageStyles }}</style>
+
+    <!-- Eventsly Scripts -->
+    <script src='/static/scripts/navScripts.js' async></script>
+
+    <!-- Font Awesome JS load for Eventsly Logo -->
+    <script src="https://kit.fontawesome.com/3f6d9190cb.js" crossorigin="anonymous"></script>
+
+    <!-- Google Font API load for Eventsly Logo -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400&display=swap" rel="stylesheet">
+
+    <!-- block for page-specific head content (stylesheets, scripts, etc) -->
+    {% block head %} {% endblock %}
+</head>
+
+<body>
+    <header>
+        <!-- .active class is applied dynamically based title variable (specified in app.py render_tempate code) -->
+        <nav role="navigation">
+            <a href="index.html" class="logo">
+                <i class="fas fa-map-marked" alt="Eventsly"></i>Eventsly
+            </a>
+            <div class='nav-menu'>
+                <a href="index.html" class="nav-item {{'active' if title == 'Eventsly'}}">Home</a>
+                <a href="Bookmarks.html" class="nav-item {{'active' if title == 'Bookmarks'}}">Bookmarks</a>
+                <a href="About.html" class="nav-item {{'active' if title == 'About Us'}}">About Us</a>
+                <a href="Contact.html" class="nav-item {{'active' if title == 'Contact'}}">Contact</a>
+                <div class="search nav-item">
+                    <span class="fa-stack">
+                        <i class="fas fa-circle fa-stack-2x" id='search-circle'></i>
+                        <i class="fa fa-search fa-stack-1x" aria-hidden="true"></i>
+                    </span>
+                </div>
+                <div class="spacer"></div>
+                <a href="login.html" class="nav-item {{'active' if title == 'Log In'}}">Log In</a>
+                <a href='register.html' class="nav-item {{'active' if title == 'Sign Up'}}">Sign Up</a>
+            </div>
+            <i class='fa fa-bars fa-inverse fa-2x' aria-hidden="true"></i>
+        </nav>
+    </header>
+
+    <main>
+        <!-- block for page specific main content -->
+        {% block main %} {% endblock %}
```

Comment:
`{% block main %} {% endblock %}` represents the page-specific content within the <main> tags of each page.